actions/lib/slack.py

```diff
@@ -46,10 +48,14 @@ def post_message_with_attachment(self, message, attachment):
 class SlackPoster(Action):
 
     def run(self, channel, user, message, **kwargs):
+        proxies = None
+        if "slack_proxy_url" in self.config and self.config["slack_proxy_url"] != "":
```

Comment:
I think a simpler formulation here might be to use `if self.config.get("slack_proxy_url")`?